File: Python/Menu.py
class Menu:
    def __init__(self, id=0, nama="", fnb="", rasa="", price=0):
        self._id = id
        self._nama = nama
        self._fnb = fnb
        self._rasa = rasa
        self._price = price

    # getter & setter id
    def get_id(self):
        return self._id

    def set_id(self, id):
        self._id = id

    # getter & setter nama
    def get_nama(self):
        return self._nama

    def set_nama(self, nama):
        self._nama = nama

    # getter & setter fnb
    def get_fnb(self):
        return self._fnb

    def set_fnb(self, fnb):
        self._fnb = fnb

    # getter & setter rasa
    def get_rasa(self):
        return self._rasa

    def set_rasa(self, rasa):
        self._rasa = rasa

    # getter & setter price
    def get_price(self):
        return self._price

    def set_price(self, price):
        self._price = price

File: Cpp/Menu.cpp
#include <bits/stdc++.h>
using namespace std;

class Menu{
    private :
        int id;
        string nama;
        string fnb;
        string rasa;
        int price;
    public :
        // constructor kosong
        Menu(){
            id=0;
            nama="";
            fnb="";
            rasa="";
            price=0;
        };
        // constructor berparameter
        Menu(int id, string nama, string fnb, string rasa, int price) {
            this->id=id;
            this->nama=nama;
            this->fnb=fnb;
            this->rasa=rasa;
            this->price=price;
        }
        // getter and setter id
        void setId(int id){this->id=id;}
        int getId() {return id;}

        // getter and setter nama
        void setNama(string nama){this->nama=nama;}
        string getNama() {return nama;}

        // getter and setter fnb
        void setFnb(string fnb){this->fnb=fnb;}
        string getFnb() {return fnb;}

        // getter and setter rasa
        void setRasa(string rasa){this->rasa=rasa;}
        string getRasa() {return rasa;}

        // getter and setter price
        void setPrice(int price){this->price=price;}
        int getPrice() {return price;}
        
        // destructor
        ~Menu(){}
};